sb.web.app/src/components/CreateArticle/StepTwo.vue

```diff
@@ -80,6 +80,28 @@ export default {
       return now.getTime() <= chosenDate.getTime() && chosenDate.getTime() <= oneMonthFromNow.getTime()
     }
   },
+  computed: {
+    locationPlaceholder () {
+      let placeholder = ''
+      if (this.chosenType.toLowerCase() === 'product') {
+        placeholder = ' where product is wanted'
+      } else {
+        placeholder = ' where service is to be performed'
+      }
+
+      return placeholder
+    },
+    pricePlaceholder () {
+      let placeholder = ''
+      if (this.savedProgress.status === 'Offer') {
+        placeholder = ' of this ' + this.chosenType.toLowerCase() + '?'
+      } else {
+        placeholder = ' you are looking to pay?'
+      }
+
+      return placeholder
+    }
+  },
```

Comment:
fwiw the i18n library has special functions for doing this kind of thing... that would let us keep all that logic in the translation side of things..

Reply:
Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific 

Also if you want to access the $i18n t method from code (as in not in the vue template) you can do it this way 
`this.$i18n.t('time.now')`

Reply:
Won't we end up chucking translations into the html code if I use named formatting? @utunga